### ISSUE TITLE
Widgets repository

### ISSUE THREAD

**Issue by jeroen** [ramnathv/htmlwidgets#55]
It might be nice if htmlwidgets would somehow support (and encourages to use) a dedicated widgets package repository (or some other module manager) to prevent CRAN from being flooded with countless packages wrapping around a contemporary JS libraries.

Maybe something as simple as `install_widget` that wraps around `devtools::install_github` to a particular github organisation would do.

**Comment by HermanSontrop** [comment]
having an HTMLWidgets repository would make it much easier to find which HTMLWidgets are available. Now you just have to be lucky to land on a page that says this and this is available. For instance, recently someone made a really cool metricsgraphics HTMLWidgets package. I know it exists, but for others this may not be so obvious. A repository (e.g. as for d3 or rCharts also exists) would help a lot.

http://www.r-bloggers.com/new-r-package-metricsgraphics/

**Comment by jeroenooms** [comment]
What is the latest policy/opinion of CRAN wrt to shipping JavaScript libraries? Do we still need to name all authors of each js file included with a package?

**Comment by jjallaire** [comment]
Based on some recent back and forth with CRAN on Shiny I'd say the answer
is yes.

Here's what I did for dygraphs (this was in response to specific feedback
regarding an incomplete specification of authors):

https://github.com/rstudio/dygraphs/blob/master/DESCRIPTION#L6-L11

**Comment by ramnathv** [comment]
@jjallaire I think we should add this to the `docs`. I will update `scaffoldWidget` to pull author names from `bower.json` so that the process can be automated for those using `bower`.

**Comment by timelyportfolio** [comment]
should we escalate this to its own issue, since it is in my opinion very important?

**Comment by ramnathv** [comment]
I would agree that it is very important. Thanks for starting a new issue @timelyportfolio
